app/moves/views/_layout.njk

```diff
@@ -22,7 +22,7 @@
 
   {% block pageHeader %}
     <header class="govuk-grid-row">
-      <div class="govuk-grid-column-two-thirds">
+      <div class="{% block mainContainerClass %}govuk-grid-column-two-thirds{% endblock %}">
```

Comment:
Just thinking about the name, we actually have a div called "main" and this block isn't actually related to that. Maybe something more appropriate to what this actually does might be better - `headerGridColumnClass`?